client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -569,6 +586,20 @@
                 </b-button>
               </b-button-group>
             </div>
+
+            <div v-if="options.showDeletedRecordsOption">
+              <div
+                class="text-nowrap font-weight-bold"
```

Comment:
If im not mistaken you can do text-nowrap on the button or the above div and then you don't need this anymore.
The font-weight-bold shouldn't be necessary